taxcalc/tests/test_growth.py

```diff
@@ -87,13 +86,13 @@ def test_target():
                         growth=growth_x)
 
     # set target
-    factor_x = {2015: {'_factor_target': [[0.04]]}}
+    factor_x = {2015: {'_factor_target': [0.04]}}
     growth_x.update_economic_growth(factor_x)
 
     target(calc_x, growth_x._factor_target, IRATES, 2015)
 
-    assert calc_x.records.BF.AGDPN[2015] == 0.04 + 0.22 + 1
-    assert calc_x.records.BF.ATXPY[2015] == 1.2572996865676191
+    assert calc_x.records.BF.AGDPN[2015] == 1.0522542782635027
```

Comment:
Will this test fail if we ever change the default `pirates` or gdp growth rates? Do we want it to fail in that scenario? 

It seems it would be helpful to test more generally that if the user says GDP growth rate should be 4%, and that is x percentage points different than the current GDP growth rate, then the aggregate variables like wages becomes x percent different the next year than they otherwise would have been (could pick a few).